hr_cae/models/hr_applicant.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Coop IT Easy SCRL fs
+#   Manuel Claeys Bouuaert <manuel@coopiteasy.be>
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class Applicant(models.Model):
+    _inherit = "hr.applicant"
+
+    title = fields.Many2one("res.partner.title")
+    job = fields.Char(string="Job", required=False)
+    job_description = fields.Text(string="Job Description", required=False)
+    equipment = fields.Text(string="Equipment", required=False)
+    type_id = fields.Many2one("hr.recruitment.degree", string="Certificate")
```

Comment:
why not "degree_id"?

Reply:
or `certificate_id`

Reply:
type_id already exists in `hr_applicant`. Here I'm just overwriting the string.

Reply:
I guess we decided to use type for certificate and that I forgot that I reluctantly agreed with it. :p